pyproject.toml

```diff
@@ -63,6 +57,22 @@ io = [
 etc = ["sparse"]
 parallel = ["dask[complete]"]
 viz = ["cartopy", "matplotlib", "nc-time-axis", "seaborn"]
+type-stubs = [
```

Comment:
I had meant to add this in a separate PR; I can do that if folks prefer / we need any discussion on it